cpp/src/Slice/Parser.cpp

```diff
@@ -742,35 +738,27 @@ namespace
         return result;
     }
 
```

Comment:
This function took a `bool namedTag` parameter which was always hard-coded.
And even worse, it took a `const string& name` which was only used some of the time based on the bool.
So I split it into 2 functions: `parseNamedCommentLine` and `parseCommentLine`. No more bool.